BrainPortal/lib/cbrain_system_checks.rb

```diff
@@ -334,6 +334,24 @@ def self.a050_check_data_provider_cache_wipe #:nodoc:
     end
   end
 
+  # prevents archiving/delete of cbrain system files and top directories, related to
+  # cache and gridshare
+  def self.a060_ensure_system_files_will_not_be_deleted #:nodoc:
+
+    #-----------------------------------------------------------------------------
+    puts "C> Updating timestamp for important system files and directories"
```

Comment:
Change message, because 'important system files and directories' is much too general. Say what is happening for real:

```
Updating DataProvider cache administrative files
```